### PR TITLE
Add localization

### PR DESCRIPTION
**Pull request by JadedCtrl**
Just followed the guide [here](https://www.haiku-os.org/community/getting-involved/translating/3rdparty/). Also includes an Esperanto translation, haha.

### CODE REVIEW

File: src/CategoryWindow.cpp [hunk 1]
```diff
@@ -18,10 +19,12 @@
 #include "CategoryListItem.h"
 #include "SQLiteManager.h"
 
+#undef B_TRANSLATION_CONTEXT
+#define B_TRANSLATION_CONTEXT "CategoryWindow"
 
 CategoryWindow::CategoryWindow()
 	:
-	BWindow(BRect(), "Category manager",B_TITLED_WINDOW,
+	BWindow(BRect(), B_TRANSLATE("Category manager"),B_TITLED_WINDOW,
```
Comment (humdingerb) [inline src/CategoryWindow.cpp:27]: Tiny code style
```suggestion
	BWindow(BRect(), B_TRANSLATE("Category manager"), B_TITLED_WINDOW,
```

Comment (JadedCtrl) [reply src/CategoryWindow.cpp:27]: Ah, thanks, I missed that! Added your changes and did SQLiteManager.cpp.

File: src/db/SQLiteManager.cpp [hunk 2]
```diff
@@ -61,33 +64,37 @@ SQLiteManager::_Initialise()
...
 
 		if (rc != SQLITE_OK ) {
 			sqlite3_free(zErrMsg);
 			sqlite3_close(db);
 
-			BAlert* alert = new BAlert("SQLITE ERROR",
-				"There was a SQLite error",
-				"OK", NULL, NULL,
+			BAlert* alert = new BAlert(B_TRANSLATE("SQLITE ERROR"),
+				B_TRANSLATE("There was a SQLite error"),
```
Comment (humdingerb) [inline src/db/SQLiteManager.cpp:96]: Tiny correction:
```suggestion
				B_TRANSLATE("There was an SQLite error."),
```
Extra points for adding the error code in brackets, like "There was an SQLite error (%code%)."
This should work to convert the int:
```
BString(error);
error << rc;
```

Comment (JadedCtrl) [reply src/db/SQLiteManager.cpp]: Thanks, just did the style fix and updated catkeys. :)

**Comment by humdingerb** [comment]
Thank you very much!
The translation at Polyglot is now open for business:
https://i18n.kacperkasper.pl/projects/28

**Comment by humdingerb** [comment]
Sorry, I totally forgot to check...: You may want to add yourself to the copyright header of all your changed files and the BAboutWindow.

**Pull request merged**